app/controllers/shoutouts_controller.rb

```diff
@@ -60,6 +60,28 @@ def create
                   .map(&:first).map { |t| t[1..-1] }.join(', ')
     )
 
+    if shoutout.tag_list.include?("london")
+      slack_client.chat_postMessage(
+        channel: Settings.slack.london,
+        text: "@#{params[:user_name]} cheered: #{message}\nSee it, and other cheers at #{root_url}!",
+        as_user: true
+      )
+    end
+
+    if shoutout.tag_list.include?("capetown")
```

Comment:
@chriscarter90 can the tag list include both London and CT at once? Just wondering, I am not familiar with the app nor the codebase.